html/js/guiutils/Tools.js

```diff
@@ -0,0 +1,46 @@
+GAMBIT.UI = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Tools object.
+    * @class
+    */
+    function Tools() {
+    }
+
+    /**
+    * Function called when New button is pressed.
+    * It creates a new Tree and draws it
+    */
+    Tools.prototype.newTree = function() {
+        var root = new GAMBIT.TREE.Node(null);
+        GAMBIT.tree = new GAMBIT.TREE.Tree(root);
+        GAMBIT.tree.updatePositions();
+        // Create a node and draw it
+        GAMBIT.tree.draw();
+    };
+
+    /**
+    * Function that swithes mode to the one specified by the button pressed
+    * @param {Button} button Button pressed that will activate mode
+    */
+    Tools.prototype.switchMode = function(button){
+        var modeToSwitch = button.getAttribute('mode');
+        // Change the class of the button to active
+        if (modeToSwitch === 'add') {
+            GAMBIT.MODE_ADD = true;
```

Comment:
We will eventually have more than 2 modes, so I think it's better to have a GAMBIT.MODE var instead, with the mode inside represented as a pseudo enum.

To define an enum in JS:

```javascript
var GAMBIT.MODES = {
  ADD: 0, 
  DELETE: 1,
...};
if (Object.freeze) {
  Object.freeze(MODES);
}
``` 

 (see: http://stackoverflow.com/a/5040502/867423 for ref)